iib/workers/tasks/opm_operations.py

```diff
@@ -1034,7 +1040,16 @@ def opm_create_empty_fbc(
 
     # Remove all the operators from the index
     set_request_state(request_id, 'in_progress', 'Removing operators from index image')
-    _opm_registry_rm(index_db_path=index_db_path, operators=operators, base_dir=temp_dir)
+    try:
+        _opm_registry_rm(index_db_path=index_db_path, operators=operators, base_dir=temp_dir)
+    except IIBError as e:
+        if 'Failed to remove operators from the index image' in str(e):
```

Comment:
This will try with permissive mode enabled for almost any failure. Is there a way to check the error message slightly more specific to package not being present?

Reply:
I've added check for more specific message. 

Do you both (@chandwanitulsi , @yashvardhannanavati) think it is sufficient or should we go down the road that Tulsi was suggesting? Unfortunately it would meant a lot of changes in our code (it is not that easy as this solution).

Reply:
@lipoja I think that depends on what happens when we add `--permissive-mode`. Does that command also remove the operators in index.db i.e non-opted in operators? If yes, then I think we should enable permissive mode by default and that would be sufficient. 

If not, then what happens if the `rm` command fails and the operators list contains few opted-in and few non-opted in operators? If this is the case, it might be more appropriate to implement what Tulsi suggested.

Reply:
I've added the comment (warning) as agreed, please check if this is sufficient.